msteams-platform/TOC.yml

```diff
@@ -826,6 +826,17 @@
       - name: Build Adaptive Card-based Loop component
         href: m365-apps/cards-loop-component.md
         displayName: loop component, weburl, metadata
+  - name: Actions in Microsoft 365
+    items:
+      - name: Overview
+        href: m365-apps/actions-in-m365.md
+        displayName: actions design
+      - name: Build Actions in Microsoft 365
+        href: m365-apps/build-actions-in-m365.md
+        displayName: build actions, retrieve action  
+      - name: Quick start guide
```

Comment:
Is it necessary to have both "Build actions" and "Quick start guide" articles? As a reader, I'd expect an article titled "Build actions in M365" to be a tutorial, and "Quick start guide" to be a fast (5-10 minute), zero-to-hero demo. As @irenewang101 mentioned, the steps of the quick start article take a longer time investment, so the title seems misleading. Could we combine these articles into a single tutorial?

Reply:
Or maybe omit the quick start guide here, and instead publish as readme for the sample